src/PrestaShopBundle/Form/Admin/Catalog/Category/AbstractCategoryType.php

```diff
@@ -122,19 +146,45 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
             ])
             ->add('active', SwitchType::class, [
+                'label' => $this->trans('Displayed', 'Admin.Global'),
+                /* IMO help here is not exactly clear. I don't understand what click on "displayed" means, maybe previously it was clickable but now isin't? */
```

Comment:
IMO help text (Click on "%displayed_label%" to index the category on your shop. where %displayed_label% = Dispalyed) for active is not exactly clear. I don't understand what click on "displayed" means, maybe previously it was clickable but now isin't?

Reply:
You're right. What do you think of replacing "Displayed" with "Enabled" and getting rid of the help text? I think the label is already quite clear on its own. 

![image](https://user-images.githubusercontent.com/70583503/103076621-6f6e2100-45ce-11eb-99ac-b6d11920ad4f.png)

Reply:
Makes sense to me. Implemented the changes.

Reply:
@JevgenijVisockij I can't see the string 'Displayed' anymore. Is it normal?